Gems/Prefab/PrefabBuilder/PrefabGroup.cpp

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "PrefabGroup.h"
```

Comment:
Repeated: Incorrect inclusion, this should be using system inclusions.

Reply:
This is how all of the private includes have been setup in PrefabBuilder. Are you asking to have all the PrefabBuilder gem updated as well?

Reply:
Hmmm, that's wrong though and shouldn't have happened.

Reply:
well it was easy enough to fix up the CMake rules to fix up these new files